executor/builder.go

```diff
@@ -525,6 +525,9 @@ func (b *executorBuilder) buildShow(v *plannercore.Show) Executor {
 	if e.Tp == ast.ShowGrants && e.User == nil {
 		e.User = e.ctx.GetSessionVars().User
 	}
+	if e.Tp == ast.ShowCreateUser && e.User == nil {
+		e.User = e.ctx.GetSessionVars().User
+	}
```

Comment:
For `show grants`, you can execute this command to see your own grants.  For `show create user` though, this syntax is not supported.  Is this required here?

Reply:
(I checked out the code, and it should be safe to just remove these 3 lines).

Reply:
I have removed these lines.